includes/specials/SpecialDeletedContributions.php

```diff
@@ -420,14 +441,14 @@ function getSubTitle( $userObj ) {
 				LogEventsList::showLogExtract(
 					$out,
 					'block',
-					$nt,
+					$userTitle,
 					'',
 					array(
 						'lim' => 1,
 						'showIfEmpty' => false,
 						'msgKey' => array(
 							'sp-contributions-blocked-notice',
-							$nt->getText() # Support GENDER in 'sp-contributions-blocked-notice'
+							$userTitle->getText() # Support GENDER in 'sp-contributions-blocked-notice'
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)